pkg/rng/entropy.go

```diff
@@ -100,66 +102,57 @@ func setAvailableTRNG() (bool, error) {
 // the entropy pool size
 // Usage:
 // go UpdateLinuxRandomness()
-func UpdateLinuxRandomness() {
+func UpdateLinuxRandomness() error {
 	good, err := setAvailableTRNG()
 	if !good {
-		log.Fatalf("Can't set available TRNG: %s", err.Error())
-		return
+		return err
```

Comment:
should this return when `err != nil` instead?

If I'm interpreting the code correctly, this should be:
```
if err != nil {
    return err
}
if !good {
    return errors.New("Could not find a good TRNG")
}
```

Reply:
yes